test/vic-v2/containers/ConfirmationPage.unit.spec.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import _ from 'lodash/fp';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import { ConfirmationPage } from '../../../src/js/vic-v2/containers/ConfirmationPage';
+
+const form = {
+  submission: {
+    response: {
+      attributes: {
+        confirmationNumber: 'V-VIC-177',
+      }
+    }
+  },
+  data: {
+    veteranFullName: {
+      first: 'Jane',
+      last: 'Doe'
+    }
+  }
+};
+
+describe('<ConfirmationPage>', () => {
+  it('should render unverified', () => {
+    const tree = shallow(
+      <ConfirmationPage form={form}/>
+    );
+
+    expect(tree.text()).to.contain('Jane  Doe');
```

Comment:
It looks like there is an extra space here?

Reply:
Yeah, the way React renders the empty middle name causes this. Browsers collapse the space when displaying.